ecosystem/node-checker/README.md

```diff
@@ -4,42 +4,135 @@ The Aptos Node Health Checker (NHC) is the reference implementation of a node he
 - **Continuous Evaluation**: As part of the AITs, Aptos Labs needs a tool to help confirm that participants are running their nodes in a way that meets our criteria. We run this tool continuously throughout each AIT to help us evaluate this.
 
-In this README we describe how to run NHC in each of the above configuration.
+In this README we describe how to run NHC for the **Operator Support** use case. NHC can reasonably be run both as an external tool as well as a sidecar process for this use case. Both are described below. For more information on how NHC works, please see [How NHC works](#how-nhc-works) below.
 
-## Running NHC
-For now, the best way to run NHC is to build and run it locally.
+## tl;dr
+While we highly recommend you read this whole README, you can get NHC working in a basic form by doing the following. This baseline configuration is for a devnet fullnode.
 
-First, check out the repo and navigate here.
+Get a baseline configuration:
+```
+cd /tmp/nhc && wget https://raw.githubusercontent.com/aptos-labs/aptos-core/main/ecosystem/node-checker/configurations/devnet_fullnode.yaml
+```
+
+Run NHC:
+```
+docker run -v /tmp/nhc:/nhc -t aptoslabs/node-checker:nightly /usr/local/bin/aptos-node-checker server run --baseline-node-config-paths /nhc/devnet_fullnode.yaml
+```
+
+Hit it with a request:
+```
+curl 'http://localhost:20121/check_node?node_url=http://mynode.mysite.com&baseline_configuration_name=devnet_fullnode'
+```
+
+## How NHC works
+Before running NHC, it is important to know at a high level how NHC works. In short, NHC runs as a service that you send http requests to in order to run a set of validations against your node. A single NHC instance can be configured to test multiple different node configurations, for example:
+
+- Validator Node running in single node testnet.
+- Public FullNode connected to devnet.
+- Validator Node connected to testnet, e.g. as part of an Aptos Incentivized Testnet.
+
+In all cases, validations are performed compared to a baseline. For example, for the second configuration above, the baseline node might be a node run by the Aptos team that demonstrates optimal performance / participation characteristics. The configuration describes where to find this node (URL + port), what evaluators (e.g. metrics checks, TPS tests, API validations, etc.) NHC should runn, what parameters to use for those evaluators, what name the configuration has, and so on. Your node will be compared to this baseline node.
+
+When you send requests to NHC, you include which configuration you want to validate your node against. This means a request to NHC might look like this:
```

Comment:
When you send requests to NHC, you must include a baseline configuration. For example, a request to NHC to use `devnet_fullnode` as baseline configuration will be like this: